_episodes/05-managing-cuda-dependencies.md

```diff
@@ -118,163 +112,84 @@ cudnn                          7.6.5       cuda9.2_0  pkgs/main
-> > nccl                         2.4.8.1      hd6f8bf8_1  conda-forge
-> > nccl                         2.5.6.1      h51cf6c1_0  conda-forge
-> > nccl                         2.5.6.1      h7cc98d6_0  conda-forge
-> > nccl                         2.5.6.1      hc6a2c23_0  conda-forge
-> > nccl                         2.5.6.1      hd6f8bf8_0  conda-forge
-> > nccl                         2.5.7.1      h51cf6c1_0  conda-forge
-> > nccl                         2.5.7.1      h7cc98d6_0  conda-forge
-> > nccl                         2.5.7.1      hc6a2c23_0  conda-forge
-> > nccl                         2.5.7.1      hd6f8bf8_0  conda-forge
-> > nccl                         2.6.4.1      h51cf6c1_0  conda-forge
-> > nccl                         2.6.4.1      h7cc98d6_0  conda-forge
-> > nccl                         2.6.4.1      hc6a2c23_0  conda-forge
-> > nccl                         2.6.4.1      hd6f8bf8_0  conda-forge
-> > ~~~
-> > {: .language-bash}
-> {: .solution} 
-{: .challenge}
-
-# Some example Conda environment files
-
-Now that you know how to figure out which versions of the various NVIDIA CUDA libraries are 
-available on which channels you are ready to write your environment.yml file. In this section I 
-will provide some example Conda environment files for PyTorch, TensorFlow, and NVIDIA RAPIDS to 
-help get you started on your next GPU data science project.
+```
+conda install pytorch torchvision torchaudio pytorch-cuda=11.7 -c pytorch -c nvidia
+```
 
-## PyTorch
+From what we've learned, we can see that the **pytorch** channel will be used at the
+first priority, then the **nvidia** channel.
 
-[PyTorch](https://pytorch.org/) is an open source machine learning library based on the Torch 
-library, used for applications such as computer vision and natural language processing. It is 
-primarily developed by Facebook’s AI Research lab. Conda is actually the 
-[recommended way](https://pytorch.org/get-started/locally/) to install PyTorch. The official 
-PyTorch binary ships with NCCL and cuDNN so it is not necessary to include these libraries in 
-your `environment.yml` file (unless some other package also needs these libraries!).
+We can check what packages have been installed using:
 
-~~~
-name: null
-
-channels:
-  - pytorch
-  - conda-forge
-  - defaults
-
-dependencies:
-  - cudatoolkit=10.1
-  - pip=20.0
-  - python=3.7
-  - pytorch=1.5
-  - torchvision=0.6
-~~~
-{: .language-yaml}
+```
+conda env export
+```
 
-> ## Check your channel priorities!
-> 
-> Also take note of the channel priorities: the official `pytorch` channel must be given priority 
-> over `conda-forge` in order to insure that the official PyTorch binaries (the ones that include 
-> NCCL and cuDNN) will be installed (otherwise you will get some unofficial version of PyTorch 
-> available on `conda-forge`).
-{: .callout}
+We might want to check this has worked:
+
+```
+python -c "import torch; print(torch.cuda.is_available())"
```

Comment:
Do you think we need to explain what this is doing?

Reply:
Good plan.